dotnet-roslyn/config/Audacia.CodeAnalysis/Base/.editorconfig

```diff
@@ -57,6 +57,9 @@ dotnet_diagnostic.ACL1013.severity = warning
 # [Naming] ACL1014: Don't include numbers in identifiers
 dotnet_diagnostic.ACL1014.severity = warning
 
+# Avoid signatures that take a bool parameter *** Suppress as covered by custom analyzer ACL1017 ***
+dotnet_diagnostic.ACL1017.severity = warning
```

Comment:
As per the [Contribtion Guidelines](https://github.com/audaciaconsulting/Audacia.CodeAnalysis/tree/master/dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers#contributing) please can you remove the changes from this file, they need to be made in a second PR after this one completes